install-dev/upgrade/sql/1.7.7.0.sql

```diff
@@ -2,3 +2,8 @@ SET SESSION sql_mode='';
 SET NAMES 'utf8';
 
 ALTER TABLE `PREFIX_search_word` MODIFY `word` VARCHAR(30) NOT NULL;
+
+/* Update wrong hook names */
+UPDATE `PREFIX_hook` SET `name` = 'actionAdministrationPageSave' WHERE `name` = 'actionAdministrationPageFormSave';
```

Comment:
You should delete instead? :thinking:

Reply:
If I do that any module hooked to that hook will have to be reinstalled.

Reply:
Right. Feel free to commit that change on this branch.

Reply:
Done!